src/components/ActionButton.jsx

```diff
@@ -12,13 +12,16 @@ const ActionButton = ({ link, text }) => {
         variant="contained"
         sx={(theme) => ({
           margin: '1rem',
-          bgcolor: 'accents.accent1',
+          bgcolor: '#1B2B7B',
```

Comment:
@hardco2020 Avoid using one-off colors instead use them from the _theme.ts_ file.

Reply:
@DivyanshuParwal  I have revised the color, could you help review? Thanks!